kfdef/kfctl_aws.yaml

```diff
@@ -6,24 +6,23 @@ metadata:
 spec:
   platform: aws
   applications:
+    - kustomizeConfig:
+        repoRef:
+          name: manifests
+          path: application/application-crds
+      name: application-crds
     - kustomizeConfig:
         overlays:
           - application
         parameters:
           - name: generateName
```

Comment:
I saw some failures with the generateName as well. The Application CRD didn't have a name field and the deployment failed as a result.

Reply:
in the config file - if it has
```
overlays:
- application
```
it will need
```
parameters:
- name: generateName
   value: <name>-
```

Reply:
Got it, thanks :) 
I must've missed that